.github/workflows/e2e-tests-ondemand.yaml

```diff
@@ -26,7 +59,7 @@ jobs:
         uses: ./.github/actions/run-e2e
         with:
           flc-namespace: dto-ocp-ondemand
```

Comment:
this is the only difference between k8s and ocp matrix. is there a way to reuse the whole action and not duplicate the steps?

Reply:
good idea, let me try. But I'm affraid if I use it as matrix it will build cartesian product and we will get:
 `flc_env: dto-k8s-1-24`, and `flc_env = dto-ocp-ondemand` together.